src/nuxt.config.js

```diff
@@ -3,22 +3,30 @@
 module.exports = {
   env: {
     SENTRY_DSN: process.env.SENTRY_DSN,
+    IS_TESTNET: process.env.IS_TESTNET,
   },
   /*
   ** Headers of the page
   */
   head: {
-    title: 'likecoin-button',
+    title: 'LikeCoin - Reinventing the Like',
```

Comment:
Need different page title for `_id`? e.g. `Send some LIKE to {likee} | LikeCoin - Reinventing the Like`

Reply:
https://github.com/likecoin/likecoin-button/blob/master/src/pages/_id/index.vue#L80
`head()` section in each page overrides this global setting